juniper_official/interface/interface-get-port-data.rule

```diff
@@ -0,0 +1,143 @@
+/*
+ * Get the port information like port name, description and the part number
+ */
+healthbot {
+    topic interfaces {
+        rule get-port-data {
+            keys port-name;
+            synopsis "Port Data Collector";
+            description "Collects port information from the device";
+            sensor components {
+                synopsis "system components open-config sensor definition";
+                description "/components open-config sensor to collect telemetry data from network device";
+                open-config {
+                    sensor-name /components/;
+                    frequency 60s;
```

Comment:
Do you require this data every 60s? This is mostly static data right?

Reply:
Hi Vijay, this is mostly static data, but any change in the hardware need to be updated before the interface data is received in storm topology.
This can be made to 3 minutes.

Let know if this ok.

Reply:
Thinking more on this..it does not matter as there are other rules which use /components at 60s.
So do please skip this. No changes required for this comment